scrooge-sbt-plugin/src/main/scala/com/twitter/ScroogeSBT.scala

```diff
@@ -179,8 +188,9 @@ object ScroogeSBT extends AutoPlugin {
       streams,
       scroogeThriftSources,
       scroogeThriftOutputFolder,
-      scroogeThriftIncludes
-    ) map { (out, sources, outputDir, inc) =>
+      scroogeThriftIncludes,
+      scroogeGenerated
```

Comment:
It will be updated as soon as some files have been generated by scroogeGen. Are you concerned about files being recreated across different runs of sbt?